beetsplug/discogs.py

```diff
@@ -302,6 +303,13 @@ def get_album_info(self, result):
         mediums = [t.medium for t in tracks]
         country = result.data.get('country')
         data_url = result.data.get('uri')
+        style = result.data.get('styles')
+        if style is None:
+            self._log.info('Style not Found')
+        elif len(style) == 0:
+            return style
```

Comment:
I expect you don't mean to return early from this function—perhaps you want `style = ''` or similar? (But the would also be the effect of the `else:` branch here.)

Reply:
I want it to return the only one style pulled fro discogs. No formatting or join is needed in the case when there is only one style tag.

Reply:
But—correct me if I'm wrong—this is in the context of the `get_album_info` function, which is supposed to return an `AlbumInfo` object. By returning early here, it's instead returning a string from this function. Am I misunderstanding something here?

Reply:
You're right. I can create a function outside of this function that creates the format for style(s)